tutorials/nnvm/nlp/from_darknet_rnn.py

```diff
@@ -0,0 +1,197 @@
+"""
+Compile Darknet Models for RNN
+==============================
+This article is an introductory tutorial to deploy darknet rnn models with NNVM.
+
+This script will run a character prediction model
+Each module consists of 3 fully-connected layers. The input layer propagates information from the
+input to the current state. The recurrent layer propagates information through time from the
+previous state to the current one.
+
+The input to the network is a 1-hot encoding of ASCII characters. We train the network to predict
+the next character in a stream of characters. The output is constrained to be a probability
+distribution using a softmax layer.
+
+Since each recurrent layer contains information about the current character and the past
+characters, it can use this context to predict the future characters in a word or phrase.
+
+All the required models and libraries will be downloaded from the internet
+by the script.
+"""
+import random
+import os
+import sys
+import time
+import urllib
+import requests
+import numpy as np
+import urllib.request as urllib2
+import tvm
+from tvm.contrib import graph_runtime
+from nnvm.testing.darknet import __darknetffi__
+import nnvm
+import nnvm.frontend.darknet
+
+MODEL_NAME = 'rnn' #Model name
+seed = 'Thus' #Seed value
+num = 1000 #Number of characters to predict
+
+# Download cfg and weights file if first time.
+CFG_NAME = MODEL_NAME + '.cfg'
+WEIGHTS_NAME = MODEL_NAME + '.weights'
+REPO_URL = 'https://github.com/dmlc/web-data/blob/master/darknet/'
+CFG_URL = REPO_URL + 'cfg/' + CFG_NAME + '?raw=true'
+WEIGHTS_URL = REPO_URL + 'weights/' + WEIGHTS_NAME + '?raw=true'
+
+def _dl_progress(count, block_size, total_size):
```

Comment:
Tutorial scripts exported to docs.tvm.ai (Ex:  https://docs.tvm.ai/tutorials/nnvm/get_started.html) .

Hence suggest to simplify the example by using existing methods.
Here we could use glueon download methods instead of rewriting.